src/components/productList/ProductList.tsx

```
@@ -1,54 +1,201 @@
+import React, { useCallback, useRef, useEffect } from "react";
```

ESLint

[error] 'useCallback' is defined but never used.

(@typescript-eslint/no-unused-vars)